Documentation/k8s-pre-reqs.md

```diff
@@ -18,6 +18,8 @@ setting up Rook in a Kubernetes cluster with Pod Security Policies enabled.
 
 ## Flexvolume Configuration
 
+**NOTE** This configuration is required when using the flex CSI driver (required for K8s 1.12 or earlier).  Going forward for K8s 1.13 and newer it is recommended to use the CSI RBD driver, making Flexvolume configuration redundant.  
```

Comment:
This documentation is clear and concise as it is. Still, you might consider:

- replacing the first "required" with "only needed" in the first sentence to emphasize the point that most users will not need these configuration steps
- "The CSI RBD driver is recommended for k8s 1.13 and newer" would be a bit briefer than what you have, and it would follow the same pattern as the first sentence (driver, then version).

The patch is accurate and complete as-is, though; no changes are needed.

Reply:
is it ` flex CSI driver` or `flex driver` ?

Reply:
It's neither I think. Good catch. It should be `FlexVolume driver`